test/modules/TestResultCoordinator/Services/TestResultReportingService.cs

```diff
@@ -71,15 +71,18 @@ async void DoWorkAsync(object state)
 
             string blobContainerUri = string.Empty;
 
-            try
+            if (Settings.Current.LogUploadEnabled)
             {
-                Uri blobContainerWriteUriForLog = await TestReportUtil.GetOrCreateBlobContainerSasUriForLogAsync(Settings.Current.StorageAccountConnectionString);
-                blobContainerUri = $"{blobContainerWriteUriForLog.Scheme}{Uri.SchemeDelimiter}{blobContainerWriteUriForLog.Authority}{blobContainerWriteUriForLog.AbsolutePath}";
-                await TestReportUtil.UploadLogsAsync(Settings.Current.IoTHubConnectionString, blobContainerWriteUriForLog, this.logger);
-            }
-            catch (Exception ex)
-            {
-                this.logger.LogError(ex, "Exception happened when uploading logs");
+                try
```

Comment:
add log to indicate log upload is run

Reply:
`TestReportUtil.UploadLogsAsync()` already prints out `Send upload logs request to edgeAgent.` I think that should be sufficient log indicator.